.circleci/config.yml

```diff
@@ -55,10 +84,44 @@ jobs:
           PROJECT="circleci/aws-code-deploy-orb-test/build"
           curl -X POST $BASE/$PROJECT?circle-token=$CIRCLECI_API_TOKEN
 
-  release:
+  slack-notify-hold:
+    machine: true
+    steps:
+      - slack/notify:
+          message: "Orbs from commit ${CIRCLE_SHA1} are ready for review/approval: https://circleci.com/workflow-run/${CIRCLE_WORKFLOW_ID}"
+          mentions: rose,eddiewebb
```

Comment:
According to Kyle's docs, this must be user IDs, not handles.  Mine is UA8FDUL8P

Reply:
ooooh ty